Build-a-Technical-Documentation-Page.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
Well done! but is too necessary to implement in your head all meta. Also, you don't have any CSS permissions 😄

Reply:
I would kindly suggest creating all meta.  It might be helpful to consider implementing all meta for a better understanding project. 👍 

Could you check the following link that helps you to understand Meta: https://www.w3schools.com/tags/tag_meta.asp

You can see a Meta example: 💯 

  <meta charset="UTF-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />